src/main/res/xml/preferences.xml

```diff
@@ -105,7 +105,7 @@
             android:summary="@string/pref_reader_config_summary"
             android:title="@string/pref_reader_config_title" />
         <EditTextPreference
-            android:defaultValue=""
+            android:defaultValue="pjpires@gmail.com"
```

Comment:
Please, remove this email address. Otherwise, I may see my inbox filled with user logs.